tests/local_testing/test_router.py

```diff
@@ -1373,6 +1375,86 @@ async def test_mistral_on_router():
 
 # asyncio.run(test_mistral_on_router())
 
+@pytest.mark.parametrize("env_base", ["OPENAI_BASE_URL", "OPENAI_API_BASE"])
+@pytest.mark.asyncio
+async def test_openai_env_on_router(monkeypatch, env_base):
+    "This tests OpenAI env variables are honored, including legacy OPENAI_API_BASE"
```

Comment:
please add a test for this under tests/litellm/litellm_main.py and make sure it's a mock test

Reply:
Just to clarify add as a mock or remove this and replace with a mock?

Reply:
went with adding one instead of removing.